### DISCUSSION TITLE
Question about profile.html

### DISCUSSION THREAD

**Discussion by MaxWellDaan** [elsa-workflows/elsa-core#663] [Q&A]
According to the instructions of the relevant documents, I successfully ran the project. Then I found a user-related avatar in the upper right corner, click on it to have a drop-down box.
![image](https://user-images.githubusercontent.com/77947252/109131076-68f65400-778d-11eb-9d98-d41aa207ac5b.png)
![image](https://user-images.githubusercontent.com/77947252/109131108-74497f80-778d-11eb-9b8d-bf043bb17028.png)
But,when I clicked, it is nothing.
Is it useful,or  I use the wrong way to open it?

**Comment by MaxWellDaan** [comment]
Does that mean that I need achieve them by myself if I want use login、logout？

**Comment by craigfowler** [comment]
I believe this is correct, to my knowledge Elsa doesn't have authentication or authorization of its own.
What you're seeing there is an extension point which isn't implemented.  You're welcome to "connect it up" to your own app's auth mechanism, or I guess to remove/hide it if you don't want it.

**Comment by sfmskywalker** [comment]
What Craig said. Going forward with the Elsa 2 release (slated for end of March), this MVC area you are using now won't be there at all. Instead, there will be a new ASP.NET Core application built with Blazor. It won't ship with auth right away, but it will be a baseline dashboard app which will continue to be worked on. It will also have more and more extensibility points.

In parallel to this app, the rewritten workflow designer itself will continue to be developed where it can ultimately be used "as-is" containing workflow management capabilities similar to the ones provided by the dashboard. The goal is for anyone to be able to simply drop a custom element in their web app (be it plain HTML/JS, React, Angular or even Blazor) and just use it. The component communicates with REST APIs exposed from the workflow server, which itself of course can be configured with auth.

Until then I recommend you simply copy the MVC Area into your own application and add any functionality you need. You can see the area as a starting point. But again, be aware that Elsa 2 will not have this area. At least not initially - I do want to provide an ASP.NET Core component that works like Hangfire's dashboard middleware down the road.
